src/kernel/bitcoinkernel_wrapper.h

```diff
@@ -969,7 +969,7 @@ class ChainView : public View<btck_Chain>
 
     BlockTreeEntry Tip() const
```

Comment:
I wonder if we should even keep these methods now. The previous calls to tip and genesis were at least thread safe (which these are no longer). I agree that it is not worth keeping them just for the sake of it, but also not sure if they still provide utility here.